handlers/handlers_test.go

```diff
@@ -67,15 +67,15 @@ func TestUnitHandlers(t *testing.T) {
 		mockConfig := config.Config{}
 
 		router := mux.NewRouter()
-		router.HandleFunc(url, Bulletin(mockConfig, mockRenderClient, mockZebedeeClient, mockArticlesApiClient))
+		router.HandleFunc(url, SixteensBulletin(mockConfig, mockRenderClient, mockZebedeeClient, mockArticlesApiClient))
```

Comment:
We will need to add here the uri var to ensure we are using the right url (rather than change the expected values to an empty string)

```suggestion
		router.HandleFunc("/sixteens/{uri:.*}", SixteensBulletin(mockConfig, mockRenderClient, mockZebedeeClient, mockArticlesApiClient))
```
And then adapt all calls to use the sixteen prefix